### ISSUE TITLE
OpenSCAD 'Save' button fails: AttributeError: unsupported keyword 'option'

### ISSUE THREAD

**Issue by jempabroni** [FreeCAD/FreeCAD#8577]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

0.20 (Release)

### Full version info

```shell
[code]
OS: Arch Linux (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.20.2.29603 (Git)
Build type: Release
Branch: makepkg
Hash: 930dd9a76203a3260b1e6256c70c1c3cad8c5cb8
Python 3.10.9, Qt 5.15.8, Coin 4.0.1, Vtk 9.1.0, OCC 7.6.3
Locale: English/United States (en_US)
[/code]
```

### Subproject(s) affected?

OpenSCAD

### Issue description

When putting any code into the Add OpenSCAD Element field, and hitting save, the command appears to fail:

```
11:51:46  Traceback (most recent call last):
11:51:46    File "/usr/lib/freecad/Mod/OpenSCAD/OpenSCADCommands.py", line 432, in saveelement
11:51:46      filename, filter = QtGui.QFileDialog.getSaveFileName(parent=self.form, caption='Open file', dir='.', filter='OpenSCAD Files (*.scad)',selectedFilter='',option=0)
11:51:46  AttributeError: PySide2.QtWidgets.QFileDialog.getSaveFileName(): unsupported keyword 'option'

```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Would you be willing to test this on the [FreeCAD development version](https://github.com/FreeCAD/FreeCAD-Bundle/releases/tag/weekly-builds) ?

**Comment by chennes** [comment]
Should be `options` (plural).

**Comment by chennes** [comment]
Thanks for the bug report. @KeithSloan I doubt this is still present in your AltImporter, but heads up just in case!

**Comment by luzpaz** [comment]
@chennes a79b397 was committed with trailing whitespace. Was that intentional ?

**Comment by chennes** [comment]
No, just an artifact of how it was constructed (and me not being able to run Black on the OpenSCAD files because it will introduce too many changes). I've addressed it in #8582, thanks.
